### PR TITLE
dashboard Ui changes

### PR DESCRIPTION
**Pull request by Akshay-p10**
Review PR and please let me know for changes ,and merge accordingly.

### CODE REVIEW

File: client/starter-app/src/app/add-money/add-money.component.scss [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
\ No newline at end of file
+.add-money{
+text-align: center;
+margin: 29px 0% ;
+padding-bottom: 50px;
+border-radius: 20px 20px 20px 20px;
+border: 1px solid #337ab7;
+
+}
+.tittle{
```
Comment (nandini-vel) [inline client/starter-app/src/app/add-money/add-money.component.scss]: title

File: client/starter-app/src/app/app.component.html [hunk 2]
```diff
@@ -1,17 +1,8 @@
...
-  <br>
-  {{httpResponse}}
-</h1> -->
 
-<app-header></app-header>
-<!-- <app-login></app-login>	 -->
+
+
+<div class="container" style="margin-top:100px;">
 <router-outlet></router-outlet>
```
Comment (nandini-vel) [inline client/starter-app/src/app/app.component.html]: Remove inline css

Comment (nandini-vel) [inline client/starter-app/src/app/app.component.html]: remove inline css

File: client/starter-app/src/app/dashboard/dashboard.component.html [hunk 3]
```diff
@@ -1,17 +1,34 @@
...
+					<span class="sr-only">Toggle navigation</span>
+				</button>
+				<div class="logo navbar-nav"><img src="assets/images/Ewallet.png">
+					<a class="navbar-brand"  href="#"><h3 style="margin:auto">MY-WALLET</h3></a></div>
+				</div>
+				<div class="navbar-collapse collapse">
+					<ul class="nav navbar-nav navbar-right">
+						<li><a><i class="fa fa-bell" aria-hidden="true"></i></a></li>
+						<li><span class="useravatar pull-right"><img src="http://lorempixel.com/100/100/people/9/"></span></li>
+					</ul>
```
Comment (nandini-vel) [inline client/starter-app/src/app/dashboard/dashboard.component.html]: Download image and put in assets folder

File: client/starter-app/src/app/dashboard/dashboard.component.scss [hunk 4]
```diff
@@ -1,40 +1,57 @@
-.dashboard{
-	text-align: center;
-	border-left: 100px solid #795548;
-	padding: 10px;
-	border-right: 100px solid #795548;
+html {
+	position: relative;
```
Comment (nandini-vel) [inline client/starter-app/src/app/dashboard/dashboard.component.scss]: wrap your css in a parent class

File: client/starter-app/src/app/passbook/passbook.component.html [hunk 5]
```diff
@@ -1,3 +1,56 @@
...
+    <td>Column content</td>
+    <td>Column content</td>
+    <td>Column content</td>
+  </tr>
+  <tr >
+    <td>4</td>
+    <td>Column content</td>
+    <td>Column content</td>
+    <td>Column content</td>
+    <td>Column content</td>
```
Comment (nandini-vel) [inline client/starter-app/src/app/passbook/passbook.component.html]: What is this supposed to be?

File: client/starter-app/src/app/send-money/send-money.component.html [hunk 6]
```diff
@@ -1,9 +1,23 @@
-<div class="sendmoney">
+<div class="sendmoney well">
+	<form class="form-inline">
```
Comment (nandini-vel) [inline client/starter-app/src/app/send-money/send-money.component.html]: what is well? give meaningful class names

File: client/starter-app/src/app/add-money/add-money.component.scss (hunk 1 above)
Comment (Akshay-p10) [inline client/starter-app/src/app/add-money/add-money.component.scss]: ok , i change this to heading

File: client/starter-app/src/app/app.component.html (hunk 2 above)
Comment (Akshay-p10) [inline client/starter-app/src/app/app.component.html]: ok Removed

File: client/starter-app/src/app/dashboard/dashboard.component.html (hunk 3 above)
Comment (Akshay-p10) [inline client/starter-app/src/app/dashboard/dashboard.component.html]: changed , now image is in assets folder

File: client/starter-app/src/app/passbook/passbook.component.html (hunk 5 above)
Comment (Akshay-p10) [inline client/starter-app/src/app/passbook/passbook.component.html]: later on we can change this , this is for ui creation . we work on this after getting data form back-end

File: client/starter-app/src/app/send-money/send-money.component.html (hunk 6 above)
Comment (Akshay-p10) [inline client/starter-app/src/app/send-money/send-money.component.html]: This well is pre-define bootstrap class used for decoration.

File: client/starter-app/src/app/dashboard/dashboard.component.scss (hunk 4 above)
Comment (Akshay-p10) [inline client/starter-app/src/app/dashboard/dashboard.component.scss]: ok i put this css inside app.component.css file.

File: client/starter-app/src/app/passbook/passbook.component.html (hunk 5 above)
Comment (nandini-vel) [inline client/starter-app/src/app/passbook/passbook.component.html]: mock the data that you will get from the backend and loop through that.

File: client/starter-app/src/app/send-money/send-money.component.html (hunk 6 above)
Comment (nandini-vel) [inline client/starter-app/src/app/send-money/send-money.component.html]: ok

**Pull request merged**